tests/helpers/non_sklearn_metrics.py

```diff
@@ -185,3 +188,31 @@ def calibration_error(
             loss += np.sum(np.nan_to_num(debias))
         loss = np.sqrt(max(loss, 0.0))
     return loss
+
+
+def d_lambda(preds: np.ndarray, target: np.ndarray, p: int = 1) -> float:
```

Comment:
this seems to be the same as `_d_lambda_compute` in such a case what is the point of testing it...
Can we get some other reference implementation, for example from https://github.com/andrewekhalel/sewar

Reply:
@Borda - Well, the whole function is almost the same as what sewar library had, except that the UQI we use is from our own library instead of from sewar, because their implementation had different kernel sizes and different filters (uniform, while we use gaussian), please note that in the research paper for UQI, they mentioned gaussian - so I feel we are more correct here. 

Also, from what I remember, sewar doesn't have tests for  D_Lambda, which means that it's not properly tested against any other implementation. I hope that answers your question, please let me know if it helps :)